build.sbt

```diff
@@ -459,32 +465,32 @@ val zio = Seq(
 
 val bcpkixJdk15Version      = "1.70"
 val bumpVersion             = "0.1.3"
-val declineVersion          = "2.3.0"
+val declineVersion          = "2.4.1"
 val directoryWatcherVersion = "0.9.10"
 val flatbuffersVersion      = "1.12.0"
-val guavaVersion            = "31.1-jre"
-val jlineVersion            = "3.21.0"
+val guavaVersion            = "32.0.0-jre"
+val jlineVersion            = "3.23.0"
 val jgitVersion             = "6.3.0.202209071007-r"
-val diffsonVersion          = "4.1.1"
+val diffsonVersion          = "4.4.0"
 val kindProjectorVersion    = "0.13.2"
-val mockitoScalaVersion     = "1.17.12"
+val mockitoScalaVersion     = "1.17.14"
 val newtypeVersion          = "0.4.4"
-val pprintVersion           = "0.7.3"
-val pureconfigVersion       = "0.17.1"
-val scalacheckVersion       = "1.16.0"
-val scalacticVersion        = "3.3.0-SNAP3"
+val pprintVersion           = "0.8.1"
+val pureconfigVersion       = "0.17.4"
+val scalacheckVersion       = "1.17.0"
+val scalacticVersion        = "3.3.0-SNAP4"
 val scalaLoggingVersion     = "3.9.4"
 val scalameterVersion       = "0.19"
-val scalatestVersion        = "3.3.0-SNAP3"
-val shapelessVersion        = "2.4.0-M1"
+val scalatestVersion        = "3.3.0-SNAP4"
+val shapelessVersion        = "2.3.10"
 val slf4jVersion            = "1.7.36"
 val slickVersion            = "3.4.1"
-val sqliteVersion           = "3.41.2.1"
+val sqliteVersion           = "3.42.0.0"
 val tikaVersion             = "2.4.1"
 val typesafeConfigVersion   = "1.4.2"
 val junitVersion            = "4.13.2"
 val junitIfVersion          = "0.11"
-val netbeansApiVersion      = "RELEASE140"
+val netbeansApiVersion      = "RELEASE180"
```

Comment:
Thank you for upgrading NetBeans version!

The new version allows us to use just `org-netbeans-modules-sampler` module without any dependencies - please remove the transitive libraries explicitly. Profiling shall still work.

Reply:
It's already done https://github.com/enso-org/enso/blob/70cdb15d0238056d23b3956d23966b8bc4b9058d/build.sbt#L701-L713